src/Angor/Avalonia/AngorApp/Composition/CompositionRoot.cs

```diff
@@ -0,0 +1,20 @@
+using AngorApp.Sections.Shell;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace AngorApp.Composition;
+
+public static class CompositionRoot
+{
+    public static IMainViewModel CreateMainViewModel(Control topLevelView)
+    {
+        var services = new ServiceCollection();
+
+        services
```

Comment:
We will also at some point have to add all the angor services, it is a good idea to do it this way, we can also add later a method `.AddAngorServices()`